### PR TITLE
add instructions to create repos on salsa

### PR DESCRIPTION
**Pull request by aviau**

### CODE REVIEW

**Comment by stapelberg** [comment]
I don’t think this is a good idea. Has there been a discussion about how we’ll approach our move to salsa? Is salsa out of beta yet?

I’d prefer it if we could have one and only one place to store repos, not two.

**Comment by aviau** [comment]
I have modified the instructions to use the "create-salsa-project" command that will be introduced in #82 . 

We can merge this after we officially move to salsa.

File: make.go [hunk 1]
```diff
@@ -821,10 +821,10 @@ func execMake(args []string) {
 	log.Printf("    git add debian && git commit -a -m 'Initial packaging'\n")
 	log.Printf("    gbp buildpackage --git-pbuilder\n")
 	log.Printf("\n")
-	log.Printf("To create the packaging git repository on alioth, use:\n")
-	log.Printf("    ssh git.debian.org \"/git/pkg-go/setup-repository %s 'Packaging for %s'\"\n", debsrc, debsrc)
+	log.Printf("To create the packaging git repository on salsa, use:\n")
+	log.Printf("    SALSA_TOKEN=\"<token here>\" dh-make-golang create-salsa-project %s", debsrc)
```
Comment (stapelberg) [inline make.go]: The SALSA_TOKEN can be removed now that create-salsa-project is done server-side.

File: make.go [hunk 2]
```diff
@@ -821,10 +821,10 @@ func execMake(args []string) {
...
 	log.Printf("\n")
 	log.Printf("Once you are happy with your packaging, push it to alioth using:\n")
-	log.Printf("    git remote set-url origin git+ssh://git.debian.org/git/pkg-go/packages/%s.git\n", debsrc)
+	log.Printf("    git remote set-url origin git@salsa.debian.org:pkg-go-team/packages/%s.git\n", debsrc)
```
Comment (stapelberg) [inline make.go]: s/pkg-go-team/go-team/

File: make.go (hunk 1 above)
Comment (aviau) [reply make.go]: done!

File: make.go (hunk 2 above)
Comment (aviau) [reply make.go]: done!

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**